tests/its/utils/pysonar-scanner-debug.py

```diff
@@ -0,0 +1,37 @@
+#
+# Sonar Scanner Python
+# Copyright (C) 2011-2024 SonarSource SA.
+# mailto:info AT sonarsource DOT com
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+#
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+"""
+This module is the entry point for debugging the pysonar-scanner in the its.
+The pysonar-scanner-debug is started with the working directory set to the root of the project, so that python
+(and pydebug by extension) can find the pysonar-scanner-debug module. However, since the scanner should be run
+from the root of the project, the working directory is changed to the project root before running the scanner.
```

Comment:
This sentence is not fully clear as you mention starting from the root of the project, then moving to the root of the project. I assume you refer to the root of the actual scanner project and the root of the project under analysis? It could probably be a bit clearer.